ringbuf/reader.go

```diff
@@ -195,3 +195,11 @@ func (r *Reader) BufferSize() int {
 func (r *Reader) Flush() error {
 	return r.poller.Flush()
 }
+
+// AvailableData returns the amount of data available to read in the ring buffer in bytes.
+// This is racy during ongoing operations but is still useful for overall information on how a ringbuffer is behaving.
+func (r *Reader) AvailableData() uint64 {
```

Comment:
Can you make this return `int` for consistency with the existing size related fields?